### DISCUSSION TITLE
Do we need to bring `AWS_IAM_CREDENTIALS_URL` support back?

### DISCUSSION THREAD

**Discussion by Xuanwo** [mozilla/sccache#1496] [Q&A] [answered]
Sccache used to support auth by:

```shell
export AWS_IAM_CREDENTIALS_URL=http://taskcluster/auth/v1/aws/s3/read-write/${bucket}/?format=iam-role-compat
```

Do we need to bring this feature back? Maybe use `SCCACHE_AWS_CREDENTIALS_URL`? AWS doesn't have env `AWS_IAM_CREDENTIALS_URL` by native, it's better to add a new one under `SCCACHE_` prefix.

**Comment by sylvestre** [comment, marked as answer]
yes please

**Reply by Xuanwo** [reply]
By adding a `SCCACHE_AWS_CREDENTIALS_URL`?

**Reply by drahnr** [reply]
I think that'd be the preferred one
